stack_elk/src/ls_telemetry.conf

```diff
@@ -30,6 +30,8 @@ input {
 		'mplstesummary', 'RootOper~MPLS_TE~Tunnels~Summary',
 		'mplstetopology', 'RootOper~MPLS_TE~Topology',
 		'mplsteautoroute', 'RootOper~MPLS_TE~AnnounceTunnelsInfo~AutorouteAnnounceTable~(?<RouterID>.*)~(?<Protocol>.*)~(?<Area>\d+)~(?<Instance>.*)']
+	      #
```

Comment:
In the sections, it might be a good idea to update the comments to indicate what is and isn't supported - e.g. under UDP indicate that only telemetry_gpb codec is supported. We may also want to update the README.md of stacks and codecs as appropriate.